yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
+    min_zoom: CASE WHEN to_number(tags->'height', '9999.99') < 10 THEN 15
```

Comment:
The `height` is a regular column, and isn't (on my DB at least) in `tags`.

Also, using `to_number` like this ignores the case where height has a unit. We handle units in [the Python transform](https://github.com/mapzen/TileStache/blob/integration-1/TileStache/Goodies/VecTiles/transform.py#L43-L87), but that's not available at this point in the process. According to my local DB, about 1/3rd of the cases where `height` is present it's in feet. We have 3 options:
1. Ignore units (current behaviour), which will make waterfalls in feet appear more important than they really are.
2. Select all waterfalls at zoom 12, and cull the smaller ones in a Python transform. Waterfalls are pretty rare features, so I don't expect this would measurably affect performance.
3. Write a SQL function to get the height in meters, effectively porting the Python one.

Which of those options is best?